relative_wallclock/relative_wallclock.py

```diff
@@ -0,0 +1,48 @@
+"""
+Adds an info-wallclock-relative command which prints the approximate wallclock time relative to the
+start of recording.
+    Usage: info-wallclock-relative
+"""
+
+import datetime
+import re
+
+import gdb
+
+from undo.debugger_extensions import (
+    debugger_utils,
+    udb,
+)
+
+
+class WallclockRelative(gdb.Command):
+    def __init__(self):
+        super().__init__("info-wallclock-relative", gdb.COMMAND_USER)
```

Comment:
Any reason for this not being `info wallclock-relative`?

Reply:
I had two reasons:

1. `info wallclock` and `info wallclock-extents` are proper commands, and I didn't want this lower quality command to look/feel the same as a proper, fully supported command. 
However...since the user has to load this addon themselves, they already know it's not fully integrated.

2. I thought it was complicated to add another subcommand to `info`. It is not!

I'll change this.